en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -0,0 +1,212 @@
+# Publishing a sample API to Service Catalog
+
+## What you'll build
+
+In this tutorial we are re-creating the healthcare API we created in the [Sending a Simple Message to a Service](../sending-a-simple-message-to-a-service) tutorial, and publish that to the Service Catalog.
+
+## Let's get started!
+
+### Step 1: Develop the integration artifacts
+
+Follow the instructions given in this section to create and configure the required artifacts.
+
+#### Create an Integration project
+
+1.  Open **WSO2 Integration Studio**.
+2.  Click **New Integration Project** in the **Getting Started** tab as shown below.
+
+    <img src="{{base_path}}/assets/img/integrate/create_project/create-integration-project.png" width="700">
+
+    This will open the <b>New Integration Project</b> dialog box.
+
+3.  Enter `ServiceCatalogSample` as the project name and select the following check boxes to create the required 
+    modules.
+    -   **Create ESB Configs**
+    -   **Create Composite Exporter**
+
+4.  Click **Finish**.
+
+    You can see the projects listed in the **Project Explorer** as shown below:
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/project-explorer-service-catalog.PNG">
+
+!!! Note
+    Note that a **resources** folder is created in the ServiceCatalogSampleConfigs project. This folder will be used to hold the swagger and metadata YAML files. These YAML files will be uploaded to the service catalog in upcoming steps.   
+
+#### Creating artifacts
+
+Create **QueryDoctorEP** and **HealthcareAPI** by following the instructions given in [Create an Endpoint](../sending-a-simple-message-to-a-service/#create-an-endpoint) and [Create a REST API](../sending-a-simple-message-to-a-service/#create-a-rest-api) sections of **Sending a Simple Message to a Service** 
+tutorial.
+
+Once the **HealthcareAPI** is created, two new files with following names will be added to the metadata directory automatically.
+
+1.  HealthcareAPI_metadata.yaml
+2.  HealthcareAPI_swagger.yaml
+
+<img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/metadata-folder-service-catalog.PNG" width="400">
+
+!!! Note
+    When registering a service in the Service Catalog, we need to provide the OpenAPI definition and metadata of the service we are registering. When creating an API, Integration Studio automatically creates these files for the user.
+
+### Step 2: Configuring meta information
+
+Open the HealthcareAPI_metadata.yaml and do the following changes.
+
+1.  Change the **description** to explain what the API does. ( Ex API to fetch doctors of a given category )
```

Comment:
```suggestion
1.  Change the **description** to explain what the API does (e.g., API to fetch doctors of a given category).
```
The abbreviation "e.g." is actually the short form for a Latin term "exempli gratia". It means "for example". So when we use this abbreviation, we should always use the correct format or it would simply not mean the same thing. Please apply for other occurrences of this in this PR.

Reply:
ack

Reply:
Fixed in a0c600c